packages/checkbox/index.tsx

```diff
@@ -45,20 +45,22 @@ interface CheckboxState {
   checked?: boolean;
   indeterminate?: boolean;
   classList: Set<string>;
-  'aria-checked': boolean;
+  'aria-checked': string;
+  disabled: boolean;
 };
 
 export class Checkbox extends React.Component<CheckboxProps, CheckboxState> {
   inputElement: React.RefObject<HTMLInputElement> = React.createRef();
-  foundation = MDCCheckboxFoundation;
+  foundation!: MDCCheckboxFoundation;
 
   constructor(props: CheckboxProps) {
     super(props);
     this.state = {
       'checked': props.checked,
       'indeterminate': props.indeterminate,
       'classList': new Set(),
-      'aria-checked': false,
+      'aria-checked': 'false',
+      'disabled': props.disabled!,
```

Comment:
what is this new state.disabled variable for?

Reply:
Adapter now has `setNativeControlDisabled: (disabled) => this.updateState('disabled', disabled),`, as I understand mdc-web now can change disabled state

Reply:
ah ok - but we don't use it in the render method.

Reply:
done